entity-framework/core/testing/index.md

```diff
@@ -1,117 +1,26 @@
 ---
-title: Testing code that uses EF Core - EF Core
-description: Different approaches to testing applications that use Entity Framework Core
-author: ajcvickers
-ms.date: 04/22/2020
+title: Overview of testing applications that use EF Core - EF Core
+description: Overview of testing applications that use Entity Framework Core
+author: roji
+ms.date: 01/17/2021
 uid: core/testing/index
 ---
-# Testing code that uses EF Core
+# Testing EF Core Applications
 
-Testing code that accesses a database requires either:
+Testing is an important concern to almost all application types - it allows you to be sure your application works correctly, and makes it instantly known if its behavior regresses in the future. Since testing may affect how your code is architected, it's highly recommended to plan for a testing early and to ensure good coverage as your application evolves. This introductory section provides a quick overview of various testing strategies for applications using EF Core.
 
-* Running queries and updates against the same database system used in production.
-* Running queries and updates against some other easier to manage database system.
-* Using test doubles or some other mechanism to avoid using a database at all.
+## Involving the database (or not)
 
-This document outlines the trade-offs involved in each of these choices and shows how EF Core can be used with each approach.
+When writing tests for your EF Core application, one basic decision you need to make is whether your tests will involve your production database system - just as your application does - or whether your tests will run against a [test double](https://martinfowler.com/bliki/TestDouble.html), which replaces your production database system. Two prominent examples of test doubles in the EF Core context are [SQLite in-memory mode](xref:core/testing/choosing-a-testing-strategy#sqlite-as-a-database-fake), and the [InMemory provider](xref:core/testing/choosing-a-testing-strategy#inmemory-as-a-database-fake). For simplicity's sake, we'll use the term *integration tests* to refer to tests which involve your production database systems, and *unit tests* to refer to tests which use a test double.
```

Comment:
"InMemory" is usually styled as "in-memory".

Reply:
Corrected everywhere.